.ado/publish.yml

```diff
@@ -133,8 +133,8 @@ jobs:
 
       - template: templates/yarn-install.yml
         parameters:
-         workingDirectory: $(Build.StagingDirectory)\bot-coordinator
-         frozenLockfile: false
+        workingDirectory: $(Build.StagingDirectory)\bot-coordinator
```

Comment:
Does this change have a runtime effect?

Reply:
It surely will. The formatter believed it was an extra space, where it was a missing one.

Some of these I'll fix manually and carefully.

I wouldn't worry about CI not catching an indentation error. The CI loops just wouldn't make sense if I get those wrong.